crowdfunding_compassion/models/crowdfunding_project.py

```diff
@@ -149,9 +149,14 @@ def _check_type(self):
     @api.constrains('presentation_video')
     def _check_presentation_video(self):
         if self.presentation_video:
-            domain = re.search(r"https?://([a-zA-Z0-9.-]+)", self.presentation_video)
-            if domain and not (domain.group(1) == 'www.youtube.com' or domain.group(1) == 'vimeo.com'):
-                raise ValidationError("The facebook link must be from the 'www.youtube.com' or 'vimeo.com' domain only.")
+            parsed_url = urlparse(self.presentation_video)
+            domain = parsed_url.netloc.lower()
+
+            if not (domain == "youtube.com" or domain.endswith(".youtube.com")
```

Comment:
Isn't the second test including the first condition. Wouldn't this give the same result (and simplify the code)?
```suggestion
            if not (domain.endswith("youtube.com")
```